docs/submit/osg-flock.md

```diff
@@ -38,32 +38,22 @@ and configure it only to forward jobs to the OSG.
 In other words, you do not need a whole HTCondor batch system just to have a local OSG access point.
 
 
-Before Starting
----------------
-Because jobs running in OSG run in a completely different environment than your local cluster,
-and require additional considerations for data movement,
-you or your campus will need to provide additional support for your users to enable them to
-properly use OSG.
-Send email to <help@opensciencegrid.org> saying that you would like to flock jobs to OSG,
-and we will [consult](#scheduling-a-planning-consultation) with you about the proper solution for your site and what
-needs to be set up at both the OSG and at your site.
+System Requirements
+-------------------
+The hardware requirement for an OSG access point depends on several factors such as number of users, 
+number of jobs and for example how I/O intensive those jobs are. Our minimum recommended configuration
+is 6 cores, 12 GB RAM and 1 TB of local disk. The hardware can be bare metal or virtual machine, but
+we do not recommend containers as these submit host are running a bunch of system services which can
+be difficult to configure in a container.
```

Comment:
```suggestion
The hardware requirement for an OSG access point depends on several factors such as number of users,
number of jobs and for example how I/O intensity of those jobs.
Our minimum recommended configuration is 6 cores, 12 GB RAM and 1 TB of local disk.
The hardware can be bare metal or virtual machine, but we do not recommend containers as these submit host are running
many system services which can be difficult to configure in a container.
```

Minor wording changes and line splits to try to reduce future diffs: https://opensciencegrid.org/technology/documentation/style-guide/#meta